apps/synapse-portal-framework/src/components/RepositoryUnderReviewAlert.tsx

```diff
@@ -0,0 +1,32 @@
+import { FullWidthAlert } from 'synapse-react-client'
+
+export type RepositoryUnderReviewAlertProps = {
+  portalSpecificDisclaimer: string
+}
+
+function RepositoryUnderReviewAlert({ portalSpecificDisclaimer }) {
+  return (
+    <>
+      <FullWidthAlert
+        sx={{ m: 0 }}
+        isGlobal={false}
+        variant={'warning'}
+        show={true}
+        title={''}
+        description={
+          'This repository is under review for potential modification in compliance with Administration directives.'
```

Comment:
IMO, the copy is unclear. Would suggest changing `Administration` to "U.S. Government", but if this is what the portal owners want, it doesn't really matter

Reply:
This text must not be modified per the NIH:
https://sagebionetworks.jira.com/browse/PORTALS-3508?focusedCommentId=244220